### ISSUE TITLE
Front-End Look and Feel

### ISSUE THREAD

**Issue by Bluebird45** [potocpav/xi_glium#2]
Every other cross-platform editor that I know of looks and behaves much the same in its different platform versions.

But Xi might, as things stand now, end up as multiple species of quite different editors with a common back-end.

If I were to build a xi front-end, efficiency/non-duplication of effort, as well as avoidance of user confusion, would cause me to make the UI look and conventions closely follow the canonical (OSX) front-end.

**Comment by potocpav** [comment]
So are you suggesting this project should mimic closely the original interface? A bit hard for me to do, when the only reference for the original is one screenshot :-)

**Comment by Bluebird45** [comment]
It seems like that screenshot might be all there is.

For the time being I'm delving into his code, trying to understand it, unfamiliar with Objective-C as I am.

**Comment by shssoichiro** [comment]
Well, the creator of the original interface had said he wanted each platform's interface to "look native", i.e. Cocoa on OS X, GTK or QT on Linux.

**Comment by Bluebird45** [comment]
Yes, I'm aware of that, so I'm seeking to follow his logic so as to apply it in another language.

**Comment by NamPNQ** [comment]
I hope it able to run in terminal and powerful like neovim, love your work
